CMakeLists.txt

```diff
@@ -46,4 +43,6 @@ if (GSL_INSTALL)
     write_basic_package_version_file(${gls_config_version} COMPATIBILITY SameMajorVersion ARCH_INDEPENDENT)
 
     install(FILES ${gls_config_version} DESTINATION ${cmake_files_install_dir})
+
+    install(FILES GSL.natvis DESTINATION ${cmake_files_install_dir})
```

Comment:
Installing the file at least gives find_package users a chance to decide what they want to do. Versus nothing which is what they currently get.